packages/@aws-cdk/aws-lambda-event-sources/lib/kafka.ts

```diff
@@ -1,9 +1,9 @@
-import { md5hash } from '@aws-cdk/core/lib/helpers-internal';
 import { ISecurityGroup, IVpc, SubnetSelection } from '@aws-cdk/aws-ec2';
 import * as iam from '@aws-cdk/aws-iam';
 import * as lambda from '@aws-cdk/aws-lambda';
 import * as secretsmanager from '@aws-cdk/aws-secretsmanager';
 import { Stack, Names } from '@aws-cdk/core';
+import { md5hash } from '@aws-cdk/core/lib/helpers-internal';
```

Comment:
Would you mind reverting this unrelated change. Thanks.

Reply:
Ah I see, thanks for explaining where this came from. 👍🏻 Probably would have been fine to leave it in then. From a PR review it's difficult to establish the source of these kind of changes (and manually checking would just take to long).

Reply:
Sorry I was in a hurry to fix it. This commit should bring us back to the beginning of this review!

Reply:
All good and thank you for being diligent. It's a tricky thing. When the same module has other change I usually don't question it.

Maybe adding a comment to the line on GitHub would help reviewers. 🤔